build.gradle

```diff
@@ -14,8 +14,10 @@ plugins {
     id "com.github.ben-manes.versions" version '0.20.0'
     id 'com.adarshr.test-logger' version '1.6.0'
     id "com.diffplug.gradle.spotless" version '3.16.0'
+    id 'com.bmuschko.docker-java-application' version '4.1.0'
```

Comment:
it seems the maintainer released the `4.2.0`. Maybe we can use it.

Reply:
7f2bb7c